src/utilities/util_verify.c

```diff
@@ -94,6 +94,9 @@ util_verify(WT_SESSION *session, int argc, char *argv[])
             else
                 return (usage());
             break;
+        case 'e':
```

Comment:
instead of 'e' how about using 'S'?

Reply:
Sure that would work too, I was thinking as `e` for `error`.

Reply:
See 762ad085118c4e6ad5bf6d21558a16c2e5cf373c